inc/notificationtargetentity.class.php

```diff
@@ -77,6 +98,11 @@ public function getTags() {
          'credit.quantity_sold'      => __('Quantity sold', 'credit'),
          'credit.enddate'            => __('End date', 'credit'),
          'credit.quantity_remaining' => __('Quantity remaining', 'credit'),
+         'credit.quantity_consumed'  => __('Quantity consumed', 'credit'),
+         'credit.child_entities'       => __('Child entities'),
+         'credit.entity'             => __('Entity'),
```

Comment:
You missed this one ;)

```suggestion
         'credit.entity'             => Entity::getTypeName(1);,
```